theme/themes/default/elements/button.variables

```diff
@@ -18,6 +18,7 @@
 @verticalPadding: @inputVerticalPadding;
 @horizontalPadding: 1.5em;
 
+
```

Comment:
@lkttyagi please try to not add blank lines in files when you don't change anything. I'd recommend to install prettier and some HTML/CSS cleaner add ons. What editor are you using?

Reply:
sure . i am using sublime text